components/Players/PlayerManager.vue

```diff
@@ -316,6 +335,11 @@ export default Vue.extend({
       }
     });
   },
+  beforeDestroy() {
```

Comment:
We should remove the event handler here

Reply:
Isn't it already in L327?

Reply:
I updated the branch a bit after @ferferga mentioned it ;)